server/apps/immich/src/api-v1/asset/asset.service.ts

```diff
@@ -140,6 +144,19 @@ export class AssetService {
     return mapAsset(updatedAsset);
   }
 
+  public async downloadLibrary(user: AuthUserDto, dto: DownloadDto) {
+    const assets = await this.assetRepository.find({
```

Comment:
Should we move this to the repository to try keeping database-related operations in a single file?

Reply:
Yeah, didn't realize this at first, I'll move the lookup to the asset repository.